libr/bin/bin.c

```diff
@@ -997,9 +1002,9 @@ static void r_bin_file_free(void /*RBinFile*/ *bf_) {
 		a->sdb_addrinfo = NULL;
 	}
 	free (a->file);
+	a->o = NULL;
 	r_list_free (a->objs);
 	r_list_free (a->xtr_data);
-	r_bin_object_free (a->o);
 	memset (a, 0, sizeof (RBinFile));
```

Comment:
let's keep that memset for now. the purpose of those memsets is to avoid some dblfree or uafs that may not exist. but it doesnt hurts for now